src/bin/bat/assets.rs

```diff
@@ -41,8 +41,12 @@ pub fn assets_from_cache_or_binary() -> Result<HighlightingAssets> {
         }
     }
 
-    Ok(HighlightingAssets::from_cache(&cache_dir)
-        .unwrap_or_else(|_| HighlightingAssets::from_binary()))
+    let custom_assets = if use_custom_assets {
+        HighlightingAssets::from_cache(&cache_dir).ok()
+    } else {
+        None
+    };
+    Ok(custom_assets.unwrap_or_else(|| HighlightingAssets::from_binary()))
```

Comment:
Could be `custom_assets.unwrap_or_else(HighlightingAssets::from_binary)` to fix the clippy suggestion.

Reply:
Dang it, I didn't see that one. Will fix before merge. Thanks for review!